Assets/AnkrSDK/Examples/UseCases/MobileUseCaseIndex.unity

```diff
@@ -2978,6 +2978,106 @@ PrefabInstance:
   m_Modification:
     m_TransformParent: {fileID: 436313774}
     m_Modifications:
```

Comment:
do not override prefabs. If it is an important change but you do not want to modify original prefab - create prefab variant.